cmd/broker/main.go

```diff
@@ -281,9 +281,9 @@ func main() {
 	fatalOnError(err, log)
 
 	gardenerNamespace := fmt.Sprintf("garden-%v", cfg.Gardener.Project)
-	gardenerAccountPool := hyperscaler.NewAccountPool(dynamicGardener, gardenerNamespace)
-	gardenerSharedPool := hyperscaler.NewSharedGardenerAccountPool(dynamicGardener, gardenerNamespace)
-	accountProvider := hyperscaler.NewAccountProvider(gardenerAccountPool, gardenerSharedPool)
+	bindingsClient := hyperscaler.NewGardenerClient(dynamicGardener, gardenerNamespace)
```

Comment:
We use (at least) two different `binding` types in KEB. 
Service Bindings and Secret Bindings. Hence `binding` term is ambiguous unless we specify which type we refer to.
I our lingo we usually use term 'secret' even if the designatum is SecretBinding not the `Secret`.